azure-pipelines.yml

```diff
@@ -24,6 +24,11 @@ jobs:
       options: --privileged
 
     steps:
+    # Temporary; this could be embedded in the Docker image, with the
+    # rest of the dependencies
+    - script: sudo apt-get update && sudo apt-get install flex bison gcc -y
```

Comment:
We could move this in the Docker image; up to @Smjert to decide how to handle this

Reply:
Yes, we can move them there.